### PR TITLE
Integration testing script for standalone pipeline with Go orchestrator code, and associated files and changes

### PR DESCRIPTION
**Pull request by kusalananda**
This PR builds on top of branch from @aaperis, which adds the necessary infrastructure to run the SDA pipeline in standalone mode using the orchestrator. His initial changes included adding a `compose-sda-standalone.yml` file for Docker compose, a separate integration testing script, `run_integration_test_standalone.sh`, and updates to the `README.md` file, all under `dev_utils`.

My changes include the following:

* Rewritten integration test script for stability and readability. The changes to this script should probably be incorporated into the other integration test scripts, or else we may want to combine all the scripts into one script with a bit of switching logic built in, depending on what it is being tested.  I have yet to look very thoroughly at the other integration testing scripts.
* Rewritten `make_certs.sh` script so that the certificate serial number is incremented if the certificates are recreated.  The most recently used serial number is stored in a file, `certs/serial.txt`.
* Added `env.orchestrate` file, the environment file for the orchestrator.

Note that the `compose-sda-standalone.yml` file included with this PR uses a _local_ `sda-pipeline` image for the `orchestrator` service as this service is not currently available in `neicnordic/sda-pipeline:latest`. The `sda-pipeline` image is built from a branch containing the changes outlined in PR #552.

### CODE REVIEW

**Review by jbygdell — APPROVED** [review]

File: dev_utils/run_integration_test_standalone.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+if [ ! -f "$infile" ]; then
+    printf 'Unable to find regular file "%s"\n' "$infile" >&2
+    exit 1
+fi
+infile=$( realpath "$infile" )
+
+cd "$(dirname "$0")" || exit
+
+# we need this certificate to be of 600 to work with db connection
+chmod 600 certs/client-key.pem
```
Comment (norling) [inline dev_utils/run_integration_test_standalone.sh:24]: Should there be a check that the certificates exist? might be good to either have a user notice to run `make_certs.sh`, or to just have the script run it if the cert is missing.

**Review by norling — APPROVED** [review]
Good stuff! :) 
Some minor things you can change if you feel like it, but nothing important

File: dev_utils/compose-sda-standalone.yml [hunk 2]
```diff
@@ -0,0 +1,274 @@
+version: "2.4"
```
[the remark this answers is not in the document]
Comment (kusalananda) [reply dev_utils/compose-sda-standalone.yml:1]: Sorted.

File: dev_utils/README.md [hunk 3]
```diff
@@ -46,6 +46,30 @@ For a complete test of the pipeline
 sh run_integration_test.sh
 ```
 
+## Starting the services in standalone mode
+
+In this case, the `orchestrator` service is used in place of the `intercept` service.  Note that the `orchestrator` service _currently_ uses a local `sda-pipeline` image as it is not yet available from `neicnordic/sda-pipeline:latest`.
```
[the remark this answers is not in the document]
Comment (kusalananda) [reply dev_utils/README.md:51]: I don't quite know how to do that. Please show.

File: dev_utils/compose-sda-standalone.yml [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+      - "9000:9000"
+      - "9001:9001"
+    volumes:
+      - ./certs/ca.pem:/root/.minio/certs/CAs/public.crt
+      - ./certs/s3.pem:/root/.minio/certs/public.crt
+      - ./certs/s3-key.pem:/root/.minio/certs/private.key
+  createbucket:
+    container_name: buckets
+    image: minio/mc
+    depends_on:
```
[the remark this answers is not in the document]
Comment (kusalananda) [reply dev_utils/compose-sda-standalone.yml:98]: Sorted

File: dev_utils/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kusalananda) [reply dev_utils/README.md:51]: Done.

**Review by blankdots — APPROVED** [review]

**Pull request merged**